src/test/java/com/github/terma/jenkins/githubprcoveragestatus/CoberturaParserTest.java

```diff
@@ -45,15 +45,15 @@ public void extractCoverageIfBranchRateIsZero() throws IOException {
         String filePath = CoberturaParserTest.class.getResource(
                 "/com/github/terma/jenkins/githubprcoveragestatus/CoberturaParserTest/cobertura-zero-branch-rate.xml").getFile();
 
-        Assert.assertEquals(0.25, new CoberturaParser().get(filePath), 0.1);
+        Assert.assertEquals(0.5, new CoberturaParser().get(filePath), 0.1);
     }
 
     @Test
     public void extractCoverageIfLineRateIsZero() throws IOException {
```

Comment:
Test name should help you to understand behaviour of class/method etc. As you want to change behaviour and ignore ```lineRate``` when it ```0``` please put that in test name, so it will be clear for people who is try to read code. For example ```ignoreLineRateIfZeroAndUseOnlyBranchRate```